### ISSUE TITLE
Bug: Opening the space advanced parameters popup is slow when an image is included in the space's description - Chrome specific

### ISSUE THREAD

**Issue by grignards** [tracim/tracim#4591]
## Description and expectations

The time to open the parameters seems linked to the file size (or image dimensions?) of the image included in the description.
It can easily take several seconds and the browser's tab is totally frozen during this delay. The CPU usage of the browser also goes up to 100% until the parameters popup is displayed. If the delay is too important the browser's displays the "a script is blocking your page" warning.

Opening the parameters is quick when an image is included via an `<img src="..." />` tag.
Including an image in a note seems to share the same behavior: opening the note in edition mode is slow after including an image in the note. It is therefore likely that all advanced editions using tinymce (space description, notes, comments, personal page in public profile) do exhibit this bug.

We should either:
- correct the open problem with images
- if the previous fix is not possible/requires too much development time: refuse "big" images/encourage the user to setup a "small" image in order to mitigate this problem.

### How to reproduce

1. Go to a space
2. Add an image in its description
3. Close the parameters dialog
4. Re-open it.

### Version information

- Tracim version: 3.8-ASF
- Web browser: All, especially slow with Chrome

________________________________________________

**Comment by gbonaspetti** [comment]
@grignards I think we have the same issue in the notes. So we agree that this fix will be applied globally to all advanced editions and not specifically to the space descriptions?

**Comment by grignards** [comment]
> @grignards I think we have the same issue in the notes. So we agree that this fix will be applied globally to all advanced editions and not specifically to the space descriptions?

Yes, I mentioned that the notes exhibit the same behavior. So, granted that the fix is not too intrusive I would expect it to be applied to all advanced editions.

**Comment by raphj** [comment]
This is a TinyMCE or Chrome bug. I tried to reproduce the bug outside Tracim with our version of tinyMCE, and the latest version (5.7.1), and the bug shows up as well on Chrome (Firefox is fine).

**Comment by raphj** [comment]
The issue can be reproduced with the `index.html` file in the following archive: [index.zip](https://github.com/tracim/tracim/files/6389482/index.zip)

The loading time on chrome takes anything from 25 seconds to 3 minutes.

Here is a profile for a 3 minute load (decompress it and load it in the Chromium's Dev Tools' performance tab): 
[profiling.zip](https://github.com/tracim/tracim/files/6389505/profiling.zip)

**Comment by raphj** [comment]
Here's a workaround: start with an empty textarea, and then append the content into the editor on initialization like this:

```
<script>
  window.onload = function () {
    tinymce.init({
      selector: 'textarea',
      language: 'en',
      menubar: false,
      resize: false,
      skin: 'lightgray',
      height:'100%',
      setup: function ($editor) {
        $editor.on('init', function (e) {
          $editor.contentDocument.body.replaceChildren(document.getElementById("content"))
        })
      }
    })
  }
</script>
<textarea></textarea>
<div id="content">
</div>

**Comment by raphj** [comment]
To test :
 - check that tinymce correctly appears everywhere it is used:
    - in the workspace description
    - in app-html
       - also try to switch between two notes
           - when edit mode is enabled
           - when it is disabled
       - test whether editing an advanced comment works, and switching between the advanced comment and the note works too
    - in the timeline
      - check that closing an app while editing a comment and then reopening it again correctly restores your text (in app html, do this for the note too)

**Comment by raphj** [comment]
Tested on branch by @grignards: https://github.com/tracim/tracim/pull/4608#issuecomment-829957219

**Comment by raphj** [comment]
There are still performance issues in comments. See https://github.com/tracim/tracim/issues/4611 to completely fix this issue.

**Comment by MariePrandstetter** [comment]
Done

**Comment by lebouquetin** [comment]
Misclick. The issue is still open (the fix was rolled back because not working)

**Comment by Skylsmoi** [comment]
It is related yes.
